public/ipython/notebook/js/celltoolbarpresets/presentation.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) IPython Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+define([
+    'jquery',
+    'notebook/js/celltoolbar',
+], function ($, celltoolbar) {
+    "use strict";
+
+    var CellToolbar = celltoolbar.CellToolbar;
+    var presentation_preset = [];
+
+    var select_width = CellToolbar.utils.select_ui_generator([
+            ["Full", "12"],
+            ["3/4", "9"],
+            ["2/4", "6"],
+            ["1/4", "3"],
+        ],
+        // setter
+        function (cell, value) {
+            // we check that the presentation namespace exist and create it if needed
+            if (cell.metadata.presentation === undefined) {
+                cell.metadata.presentation = {};
```

Comment:
no matter, the `presentation` level is reasonable. Of course, it's sad we have to hack that deep in cell.js :-S.

but since it works, let keep that!